_pages/rfc-pairing.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pageminimal
+title: "RFC [r] - WeAllJS #pairing"
+permalink: /rfc-pairing
+---
+
+# RFC [r] - WeAllJS #pairing
+
+[ratified 2016-09-17]
+{: text-italic}
+
+Learning is cool! Programming is hard! Learning with someone else is often way more productive than trying to learn alone.
+We All JS  strives to be a community where people learning something new, no matter their level of skill can find support.
+Here at We All JS, we recommend two different forms of cooperative programming.
+
+## Code Review
+Code review is an extremely useful medium for cooperative learning. A more experienced person code reviewing code from a
+newcomer might find that new techniques and technologies have emerged that are out of their expertise. Newcomers can benefit
+from the knowledge of individuals who have put their experience to the test.
+
+You are free to ask for code review in anyway you like,  but here is our suggestion
+* Push your code to a Github repository.
+* Open a feature branch, and start coding away.
+* Push the feature branch upstream, and open a pull request from that feature branch into master.  To do this, go to your
+project's homepage, and there should be a button that says **New pull request**. Click this button. Select master as your base
+branch, and your feature branch as the compare branch. Then, click the **Create pull request** button.
+* This should bring you to a page where you can write a description. Describe your problem, and what your pull request is aiming to
+solve. Indicate to reviewers the files and sections of code that you would like review on. After you are done, click **Create pull
+request.**
+* You should now be at a pull request page. Copy the link and paste it into the channel, asking for review. Indicate the languages and
+frameworks you are using.
+* Reviewers should go into the page, and select the Files changed tab. In this tab, one can see the diffs of the commits.
+Hovering your mouse over the diff should display a plus sign. Clicking on this plus sign will allow the reviewer to make line notes
+* As reviews come in, respond to code review by making appropriate code changes. You are also able to reply to inline comment by
+commenting on the same line. Talk it out, and don't be afraid to ask questions either on the pull request page, or on
+the pairing page. Push up any commits you make in the process.
```

Comment:
Should this be pairing channel? As it is, makes it unclear whether there is another webpage for pairing.